pandas/core/internals/blocks.py

```diff
@@ -603,7 +602,7 @@ def _astype(self, dtype, copy=False, errors='raise', values=None,
                     values = self.values.astype(dtype)
                 else:
                     if issubclass(dtype.type,
-                                  (compat.text_type, compat.string_types)):
+                                  (str, str)):
```

Comment:
I suppose this could just be a single str

Reply:
Oops, thought I fixed all of those. Nice catch!  I'll give this another once over to see if there are other instances of this beyond your additional comments.

Reply:
done